### ISSUE TITLE
Attempting to Reply Just Crashes Client

### ISSUE THREAD

**Issue by dorkbuttt** [chucker/Mastonaut#144]
Whenever I press the button to reply -- either in the post column on the far right or on the post itself in whatever column it is, my client immediately crashes.

[crash.txt](https://github.com/chucker/Mastonaut/files/12009696/crash.txt)

**Comment by chucker** [comment]
Hey, I see in your crash log that you're running a macOS 14 beta.

Indeed, I can confirm that opening the status composer crashes. Will investigate. But in the meantime, do you have macOS 13 somewhere still? :)

**Comment by dorkbuttt** [comment]
I can boot one up in a Parallels virtual machine later today.

**Comment by chucker** [comment]
So, the crash seems to be here:

```
	var currentAccount: AuthorizedAccount?
	{
		didSet
		{
			currentAccountObservations.removeAll()

			if let currentAccount = currentAccount
			{
				let accountUUID = currentAccount.uuid
				client = Client.create(for: currentAccount)
				authorAvatarView.image = #imageLiteral(resourceName: "missing.png") // crash
```

Replacing that with just "missing" seems to do the trick.

Apparently, that was always wrong (or has been wrong for a while, anyway), so my guess is macOS 14 Sonoma handles unknown images more strictly?

Anyway, I guess I'll have to sift through all code to see what other references are wrong…

**Comment by chucker** [comment]
>Apparently, that was always wrong (or has been wrong for a while, anyway), so my guess is macOS 14 Sonoma handles unknown images more strictly?

More nerdy details on this over at https://norden.social/@chucker/110709691896547050. I've filed a bug report with Apple, since I'm not sure this is an intended change.

But the workaround is easy enough, and the code seems cleaner that way anyway, so…

**Comment by chucker** [comment]
Fixed in 1.8: https://github.com/chucker/Mastonaut/releases/tag/app-1.8.0
